modules/administration-guide/examples/snip_che-create-monitoring-namespace.adoc

```diff
@@ -0,0 +1,4 @@
+[subs="+attributes,quotes"]
```

Comment:
```suggestion
[source,terminal,subs="+attributes,quotes"]
```

Reply:
Sounds great, I've added `source,terminal,` too all of the instances of `$` in the files in this PR